docs/puzzles/day5.md

```diff
@@ -2,12 +2,206 @@
 sidebar_position: 5
 ---
 
+by [@tgodzik](https://github.com/tgodzik)
+
```

Comment:
Maybe you need to move that after the title `# Day 5: Hydrothermal Venture`.

Reply:
Done!